src/cltk/stops/cop.py

```diff
@@ -0,0 +1,329 @@
+"""
+This list is adapted from https://github.com/computationalstylistics/tidystopwords
+which in turn is based on UniversalDependencies treebanks.
+"""
+
+STOPS = [
+	"ϣⲁ", # lemma: ϣⲁ, UD_pos: ADP
+	"ϣⲁⲣⲟ", # lemma: ϣⲁ, UD_pos: ADP
+	"ϣⲁⲣⲱ", # lemma: ϣⲁ, UD_pos: ADP
+	"ϩⲁ", # lemma: ϩⲁ, UD_pos: ADP
+	"ϩⲁϩⲧⲏ", # lemma: ϩⲁϩⲧⲛ, UD_pos: ADP
+	"ϩⲁⲣⲁⲧ", # lemma: ϩⲁⲣⲁⲧ, UD_pos: ADP
+	"ϩⲁⲣⲟ", # lemma: ϩⲁ, UD_pos: ADP
+	"ϩⲁⲣⲱ", # lemma: ϩⲁ, UD_pos: ADP
+	"ϩⲁⲧⲉ", # lemma: ϩⲁⲧⲉ, UD_pos: ADP
+	"ϩⲁⲧⲉ", # lemma: ϩⲁϩⲧⲛ, UD_pos: ADP
+	"ϩⲁⲧⲛ", # lemma: ϩⲁⲧⲛ, UD_pos: ADP
+	"ϩⲓ", # lemma: ϩⲓ, UD_pos: ADP
+	"ϩⲓϫⲙ", # lemma: ϩⲓϫⲛ, UD_pos: ADP
+	"ϩⲓϫⲛ", # lemma: ϩⲓϫⲛ, UD_pos: ADP
+	"ϩⲓϫⲱ", # lemma: ϩⲓϫⲛ, UD_pos: ADP
+	"ϩⲓⲣⲙ", # lemma: ϩⲓⲣⲛ, UD_pos: ADP
+	"ϩⲓⲧⲙ", # lemma: ϩⲓⲧⲛ, UD_pos: ADP
+	"ϩⲓⲧⲛ", # lemma: ϩⲓⲧⲛ, UD_pos: ADP
+	"ϩⲓⲧⲟⲟⲧ", # lemma: ϩⲓⲧⲟⲟⲧ, UD_pos: ADP
+	"ϩⲓⲧⲟⲟⲧ", # lemma: ϩⲓⲧⲛ, UD_pos: ADP
+	"ϩⲓⲧⲟⲩⲱ", # lemma: ϩⲓⲧⲟⲩⲛ, UD_pos: ADP
+	"ϩⲓⲱ", # lemma: ϩⲓ, UD_pos: ADP
+	"ϩⲓⲱⲱ", # lemma: ϩⲓ, UD_pos: ADP
+	"ϩⲙ", # lemma: ϩⲛ, UD_pos: ADP
+	"ϩⲙ", # lemma: ϩⲙ, UD_pos: ADP
+	"ϩⲛ", # lemma: ϩⲛ, UD_pos: ADP
+	"ϫⲓⲛ", # lemma: ϫⲓⲛ, UD_pos: ADP
+	"ⲁϫⲛ", # lemma: ⲁϫⲛ, UD_pos: ADP
+	"ⲉ", # lemma: ⲉ, UD_pos: ADP
+	"ⲉ", # lemma: ⲉⲣⲉ, UD_pos: ADP
+	"ⲉ", # lemma: ⲉⲣⲉ_ⲛⲧⲟⲟⲩ, UD_pos: ADP
```

Comment:
I noticed that these three tokens have the same word_from but different lemmas.
This needs further inspection

Reply:
Good observation. Also: Whatever these words mean, someone has thought about inflecting them for us.